app/src/auth.ts

```diff
@@ -76,9 +76,6 @@ export async function refresh({ navigate }: LogoutOptions = { navigate: true }):
 	}
 
 	try {
-		// Delete the token header if it still exists
-		delete api.defaults.headers.Authorization;
-
 		const response = await api.post('/auth/refresh');
```

Comment:
The reason it was there is because you can't request a /refresh with an access token (as it would still validate that access token). We should update this call to explicitly nullify the authorization header for this particular call:

```suggestion
		const response = await api.post('/auth/refresh', { headers: { 'Authorization': undefined }});
```

Reply:
Thank you! @rijkvanzanten